talk/owt/sdk/base/stream.cc

```diff
@@ -752,6 +752,14 @@ size_t QuicStream::ReadableBytes() const {
   }
 }
 
+uint64_t QuicStream::BufferedDataBytes() const {
+  if (quic_stream_ && !fin_read_) {
```

Comment:
From my understanding, a stream may still send data even fin is received.

Reply:
done.